app/src/main/java/umc/cozymate/ui/cozy_bot/CozyBotFragment.kt

```
@@ -96,6 +110,14 @@ class CozyBotFragment : Fragment() {
         if (roomId != 0) {
             setName()
             viewLifecycleOwner.lifecycleScope.launch {
+                try {
+                    viewModel.fetchRoomInfo()
+                } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)